websockets/server.py

```diff
@@ -232,7 +232,9 @@ def get_response_status(self, set_header):
 
         This method may be overridden to check the request headers and set a
         different status, for example to authenticate the request and return
-        ``HTTPStatus.UNAUTHORIZED`` or ``HTTPStatus.FORBIDDEN``.
+        ``HTTPStatus.UNAUTHORIZED`` or ``HTTPStatus.FORBIDDEN``. The
+        current request path can be accessed from within this method
+        as ``self.path``.
```

Comment:
It may be useful to describe how to access headers (``request_headers``, ``raw_request_headers``) as well, which doesn't quite belong to a docstring.

Perhaps it would be best to add a section to the docs about customizing the handshake process and give all the details there. What do you think?

(I also need to look at the other related issue.)